src/Micronetes.Hosting/Infrastructure/ProcessUtil.cs

```diff
@@ -72,9 +73,9 @@ internal static class ProcessUtil
                 {
                     if (e.Data != null)
                     {
-                        if (outputDataReceived != null)
```

Comment:
I think the logic should put everything in output data unless error data is specified. By default there's no value is splitting the streams here.